scripts/setup.sh

```diff
@@ -122,11 +122,13 @@ kubectl delete storageclass standard
 kubectl create -f ../kube/storageclass.yaml
 helm init --upgrade --wait
 
-if [ "$CREATE_EIRINI_NAMESPACE" = true ] ; then
+if [ "$ENABLE_EIRINI" = true ] ; then
   kubectl create namespace eirini
```

Comment:
They can't be tied together, it is separate now just because between the eirini-bosh release changes we do create the namespace now ( and in the future we can drop the creation at all). 

In the new eirini bosh release(>0.0.4)  certs are copied in the namespace if the namespace didn't existed before ( so to be idempotent, we should copy the certs required for the uploader)

Reply:
Oh wait true, nevermind my prior comment, this is ok. If we create it always should be ok as well